Terminal.Gui/View/Layout/ViewLayout.cs

```diff
@@ -191,15 +191,15 @@ void ThicknessChangedHandler (object sender, EventArgs e)
 			Margin.ThicknessChanged -= ThicknessChangedHandler;
 			Margin.Dispose ();
 		}
-		Margin = new Frame () { Id = "Margin", Thickness = new Thickness (0) };
+		Margin = new Frame () { Thickness = new Thickness (0) };
```

Comment:
In the future if `Margin` has a different behavior than `Frame` then you will to sub-class it.

Reply:
Yessir. 

I already have one difference: Margin uses SuperView color scheme. Border and padding uses View.